Initial commit

yaroslav-furhala opened the pull request:
- [DEMO LINK](https://yaroslav-furhala.github.io/layout_stop-watch/)

SanyaBratashchuk commented on src/styles/block/arrows.scss:
```diff
@@ -0,0 +1,64 @@
+    animation: round 86400s steps(12, end) infinite normal 0s backwards;
+  }
+
+  @keyframes round {
+    from {
+      transform: rotate(0);
+    }
+
+    to {
+      transform: rotate(360deg);
+    }
+  }
```
move keyframes to the seperate file

SanyaBratashchuk commented on src/styles/block/arrows.scss:
```diff
@@ -0,0 +1,64 @@
+  }
+
+  &__second {
+    position: absolute;
+    top: 0;
+    right: $half;
+    width: $thickness;
+    height: $half;
+    background-color: $center-color;
+    transform-origin: center bottom;
+    animation: round 60s steps(60, end) infinite normal 0s backwards;
+  }
+
+  &__minute {
+    position: absolute;
+    top: 0;
+    right: $half;
+    width: minuteThickness($thickness);
+    height: $half;
+    background-color: $center-color;
+    transform-origin: center bottom;
+    animation: round 3600s steps(60, end) infinite normal 0s backwards;
```
too much repetitive styles

SanyaBratashchuk commented on src/styles/utils/_functions.scss:
```diff
@@ -0,0 +1,9 @@
+@import "../utils/variables";
+
+@function minuteThickness($thickness) {
+  @return ($thickness * 2);
+}
+
+@function hourThickness($thickness) {
+  @return ($thickness * 3);
+}
```
what is 2 and 3? 
```suggestion
@function getThickness($multiplier) {
  @return ($thickness * $multiplier);
}
```

yaroslav-furhala commented:
1.  (move keyframes to the seperate file) - Done!
2.  (too much repetitive styles) - I unified them and now, in order to change the diameter of the watch, you need to change only two parameters, the main diameter and the thickness of the hand, to achieve this, I needed such a unification!
3.  (what is 2 and 3?) - 2 and 3 are three multipliers, i.e. the minute hand is twice as thick as the second hand and the hour hand is three times as thick as the second hand. Therefore, I added these multipliers (and I reworked the function according to your wishes, added two multipliers in the variable for the thickness of the minute and hour hands), I set such a proportionality so that the thickness changes proportionally for all three hands!
